### ISSUE TITLE
Feature Request - volume and other volume

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#7791]
**Is your feature request related to a problem? Please describe.**

https://arctos.database.museum/info/ctDocumentation.cfm?table=cttissue_volume_units has a lot of overlap with https://arctos.database.museum/info/ctDocumentation.cfm?table=ctvolume_units. It should be renamed, or go away, or something.

**Describe what you're trying to accomplish** 

Not have a bunch of ways of saying the same thing, and maybe avoid [misleading names](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type#remaining_volume) in the process.

**Describe the solution you'd like**

Consistency

**Describe alternatives you've considered**

Chaos and undiscoverability.

**Additional context**

Recently https://github.com/ArctosDB/arctos/issues/7783#issuecomment-2105214665 - if something exists someone will try to use it.

See also https://github.com/ArctosDB/arctos/issues/5718 https://github.com/ArctosDB/arctos/issues/5752 https://github.com/ArctosDB/arctos/issues/3648 etc.

https://github.com/ArctosDB/arctos/issues/7414#issuecomment-1947423308 was the decision to not address this

**Priority**

This has been causing problems and confusion for years and is actively trying to grow, seems pretty high to me.

**Comment by dustymc** [comment]
This is still causing problems, https://github.com/ArctosDB/data-migration/issues/1517#issuecomment-2288808176

Strongly suggest https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type#remaining_volume go away or be disambiguated.

Replacements are

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type#volume
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type#weight

Adding to AWG discussions, this is actively causing problems and needs addressed in some way.

**Comment by campmlc** [comment]
I really do not like having to lose the implicit information in the term "remaining".  For consumable tissue loans, we need to be able to distinguish the volume or mass of what is left of a sample after is subsampled for loan. Neither "weight"  or "volume" clearly indicates how these are distinct from "remaining volume" other than just some vague awareness that there is a difference in date.
I also object to calling 5 mg a "weight". This is mass - there is no appreciable "weight" here. Weight is a lay term that explictly includes the effect of gravity on mass and is something that would be used for large containers or fossils in crates. 

These terms are for curatorial usage. The general public does not need to know this information. Why lose important context by dumbing down the vocabulary and inserting potential confusion that can cause problems for data entry and curatorial workflows? My students and staff have voiced the same concerns, and for this reason we are still using remaining volume.
So question: is this issue causing problems just because of the units code table? Do we have to have units separated by mass or volume units, or can they share a code table? 
My preference would be to convert to "remaining amount" as previously discussed, and use a units code table that allows both types of units. Happy for "weight" and "volume" attributes to still exist if some collections need them. Again - denormalization is not an issue here, as these are internal terms used only to make life easier for collection managers.

**Comment by mkoo** [comment]
These terms were discussed A LOT in CT meetings. A decision was made in CT to make the term more generic and useful across all collection types especially when coupled with a date of measuring or with noting subsamples. I am not seeing a new argument being presented for ```remaining volume``` and the original issue has nothing to do with units

>I also object to calling 5 mg a "weight". This is mass ....
Separate issue which is also addressable with units. If a type is missing please file an issue but off topic here.

BTW CT meets tomorrow!

**Comment by mkoo** [comment]
On the agenda for next CT meeting but essentially  https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type needs a definition and we cannot continue with a table of mixed units

**Comment by dustymc** [comment]
Here's who's using https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type#remaining_volume

```
 guid_prefix | count 
-------------+-------
 MSB:Bird    |    68
 MSB:Herp    |    85
 MSB:Mamm    |  1001
 MVZ:Bird    |     1
 MVZ:Herp    |    94
 MVZ:Mamm    |    10
```

**Comment by mkoo** [comment]
I'll get back to you!

**Comment by dustymc** [comment]
Thanks @mkoo done, yes someone ALWAYS gets lost in any denormalization.

Here's your old data:

[temp_rvol.csv.zip](https://github.com/user-attachments/files/16954765/temp_rvol.csv.zip)

and a summary of what's left:

```

 guid_prefix | count 
-------------+-------
 MSB:Bird    |    68
 MSB:Herp    |    85
 MSB:Mamm    |  1001
```

**Comment by campmlc** [comment]
@dustymc can you automagic these into the volume and (ugh) "weight" part attributes based on their units, with a note in remarks "converted from remaining volume"?

**Comment by dustymc** [comment]
@campmlc absolutely. Here's your pre-update data:

[temp_rvol(1).csv.zip](https://github.com/user-attachments/files/16955066/temp_rvol.1.csv.zip)


done and cleaned, thanks!
